tests/DecodeChunkedStreamTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace React\Tests\HttpClient;
+
+use Exception;
+use React\HttpClient\ChunkedStreamDecoder;
+use React\Stream\ThroughStream;
+
+class DecodeChunkedStreamTest extends TestCase
+{
+    public function provideChunkedEncoding()
+    {
+        return [
+            [
```

Comment:
Yeah should have done that from the start, now I have to figure out which is which 👍